### PR TITLE
jp.cyberlink.com

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/104716

<details><summary>Screenshot:</summary>

Снимок экрана 2022-01-17 в 18 23 02

Снимок экрана 2022-01-17 в 18 27 23

Снимок экрана 2022-01-17 в 18 23 22

Снимок экрана 2022-01-17 в 18 31 15

</details><br/>

### CODE REVIEW

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -37,6 +37,7 @@ cyzo.com##.pab30
 so-zou.jp###g-ad
 wezz-y.com###adEntryRectangleBottom
 wezz-y.com###adSideRectangleA
+jp.cyberlink.com###faceme_a
```
Comment (Yuki2718) [inline JapaneseFilter/sections/specific.txt:40]: Why do you block this? It's their product. I won't block even in Annoyances.

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -60,6 +60,7 @@ bookmyshow.com##.bcBRAA
 bookmyshow.com##div[class="df-au df-av df-cq"]
 jibunde-esute.com##.p-snsShare
 exkavator.ru##.social__wrap
+jp.cyberlink.com##.news_nav > :not([class="backUrl"])
```
Comment (Yuki2718) [inline SocialFilter/sections/specific.txt:63]: I don't see this in the page.

File: AnnoyancesFilter/sections/popups.txt [hunk 3]
```diff
@@ -127,6 +127,9 @@ cuevana3.io###mdl-ads
 mine.toyokeizai.net#$#.free-modal { display: none!important; }
 mine.toyokeizai.net#$#.is-scroll-disabled { overflow: auto!important; }
 kanaloco.jp###rt_popup
+jp.cyberlink.com#$##macDetectPopupModal { display: none !important; }
+jp.cyberlink.com#$#.modal-backdrop { display: none !important; }
+jp.cyberlink.com#$#.modal-open { overflow: auto !important; }
```
Comment (Yuki2718) [inline AnnoyancesFilter/sections/popups.txt:132]: Are these for Mac? I don't see.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
You can drop jp subdomain from all the rules. The site is multi-national e.g. `https://tw.cyberlink.com/index_zh_TW.html`

File: SocialFilter/sections/specific.txt (hunk 2 above)
Comment (jellizaveta) [reply SocialFilter/sections/specific.txt:63]: https://jp.cyberlink.com/jpn/press_room/view_4797.html

<details><summary>Screenshot:</summary>

Снимок экрана 2022-01-17 в 18 56 36

</details><br/>

Comment (Yuki2718) [reply SocialFilter/sections/specific.txt:63]: Okay, nice pick!

**Comment by Yuki2718** [comment]
Maybe `||cyberlink.com^$removeparam=affid` can be added: `https://jp.cyberlink.com/blog/youtube/261/guide-to-audio-for-youtube?affid=2581_1572_730_49856_0_JPN`

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**